### PR TITLE
all: only run Windows Travis builds on cron, not during pre-commit checks

### LINKED ISSUE #862: all: Travis takes 12m to run, want 5m
Even after splitting wire out into its own github repository, it's taking an excessive amount of time for our tests to run.

### PR DESCRIPTION
**Pull request by ijt**
Fixes #862.

### CODE REVIEW

**Comment by ijt** [comment]
The Travis check took 5m to run. https://github.com/google/go-cloud/pull/888/checks?check_run_id=37705476

**Comment by ijt** [comment]
Reassigning to @jba since @vangent is TAing a Go class today.

File: internal/testing/runchecks.sh [hunk 1]
```diff
@@ -28,6 +28,11 @@ fi
 
 result=0
 
+if [[ "$TRAVIS_OS_NAME" == "windows" && "$TRAVIS_EVENT_TYPE" != "cron" ]]; then
```
Comment (vangent) [inline internal/testing/runchecks.sh]: Could you make this `== "pullrequest"` or whatever to make it opt-out instead of opt-in?

Comment (ijt) [reply internal/testing/runchecks.sh]: Done.

**Pull request merged**